src/styles/card.scss

```diff
@@ -0,0 +1,110 @@
+.card {
+  width: 200px;
+  padding: 15px 14px;
+
```

Comment:
redundant space
please take a look on all redundant spaces in css files and remove them